### DISCUSSION TITLE
Why the Mental Health of Liberal Girls Sank First and Fastest by Jonathan Haidt

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#356] [General]
https://jonathanhaidt.substack.com/p/mental-health-liberal-girls

## Excerpts

> I am increasingly convinced that there are tremendously negative long-term consequences, especially to young people, coming from this reliance on the language of harm and accusations that things one finds offensive are “deeply problematic” or even violent. Just about everything researchers understand about resilience and mental well-being suggests that people who feel like they are the chief architects of their own life — to mix metaphors, that they captain their own ship, not that they are simply being tossed around by an uncontrollable ocean — are vastly better off than people whose default position is victimization, hurt, and a sense that life simply happens to them and they have no control over their response.

### Depression scores are trending up since 2012 and most strongly for Liberals and for Women 

![image](https://user-images.githubusercontent.com/180658/224604385-cfeb885d-3d4e-4571-aadb-71ce9a45fe2c.png)

### Rising sense of external locus of control

![image](https://user-images.githubusercontent.com/180658/224604445-73496fdf-c091-4830-abd0-970761facf9a.png)

**Comment by theo-cox** [comment]
Thanks for sharing @rufuspollock , this was a really interesting read. I think my key comments as as follows, but basically all amount to a position that for this debate to constructively move forward we need more of a "both and" approach from all sides, including sources like this blog:

1) I agree that the cancel/trigger warning culture has gone ridiculously too far in many cases (the take a stab example was wild) AND:
- a) I think it's ill advised to throw the baby out with the bathwater and lump that in with in my view far more legitimate things such as race/diversity trainings which encourage students to engage with the reality of structural oppression (all this stuff gets run unhelpfully together as part of the same bundle imo) and; 
- b) more broadly, getting hung up on these more  extreme cases can hinder proper engagement with _why_ people think and feel these things and the search for shared understanding and support rather than just condemnation. Focus on extremes makes it harder to find the signal amongst the noise on each side.

2) while I agree there's a lot of harm that comes from a sense of lacking agency, I also think Haidt etc al (and critics more generally) have been way too quick to dismiss the reality of the systemic situation young people face. Of course let's build resilience etc but I also think it's unfair and disingenuous to basically take the line that it's all a matter of personal deficiency when young people  do face the most disempowering systemic context in recent history. Obama and gay marriage do not in any way compensate for the climate crisis and fallout from 2008, and the continuing reality of systematic oppression along racial, gender and class based grounds etc and we need to be upfront about that. I think reducing out the systemic context (and engagement with it, which I'd argue is done more by "liberals" and which has increased in later generations even if systemic oppression has always been there) also tends to perpetuate the same worldview of western individualism which we all agree needs to shift dramatically. For what I view as a far more compelling way of reconciling systemic context and agency I'd recommend taking a look at liberation psychology which I've recently come across (https://www.bps.org.uk/psychologist/liberation-psychology-history-future).

3) Similarly I do sigh every time I read interpretations of stuff around language/"free speech" as people "just being sensitive and thinking words can hurt them". Again, it's gone too far but there is also a reality that e.g. trans people face massively more _actual_ violence than any other group in society, and pretending that social narratives and discourses about their identity don't contribute to this is again in my view just unrigorous apart from anything. Doesn't mean I endorse shutting down all debate or blanket language bans etc but again to the larger point, let's engage with the need underlying this strategy we might not agree with and work together on how to meet it, rather than just writing it all of as snowflakery and missing the material reality underpinning it.

4) Relatedly, I'd like to see a lot more focused qualitative engagement with members of Gen Z about their view of the world, to support the hypotheses and prescriptions drawn from the data. I think a lot of richness is otherwise potentially lost (from my understanding this is sort of what warm data is getting at: https://batesoninstitute.org/warm-data/#:~:text=WHAT%20IS%20WARM%20DATA%3F,data%2C%20(cold%20data), and again I think the approach of making all these big claims _about_ groups without speaking _to_ them both risks missing nuance and is just unhelpful re finding resolutions. I also note that it's often very privileged older people casting big aspersions downwards in a way I think risks oversimplification from the insulation of perspective, not to mention recreating adversarial dynamics which lead to deadlock.
